plugin/plugin.go

```diff
@@ -279,71 +296,77 @@ func (p *AzurePlugin) ListHosts(ctx context.Context, req *pb.ListHostsRequest) (
 	// Anyways, now we find VM details for those resources
 	vmToIfaceMap := make(map[string][]network.Interface)
 	{
+		var virtualMachineTypeSb strings.Builder
+		virtualMachineTypeSb.WriteString(constMsComputeService)
+		virtualMachineTypeSb.WriteString("/")
+		virtualMachineTypeSb.WriteString(constVirtualMachinesResource)
```

Comment:
We can just write this as
```suggestion
		virtualMachineType := constMsComputeService + "/" + constVirtualMachinesResource
```

Reply:
Old Java habits die hard...